packages/flutter/lib/src/material/slider.dart

```diff
@@ -212,41 +214,55 @@ class Slider extends StatefulWidget {
 
 class _SliderState extends State<Slider> with TickerProviderStateMixin {
   static const Duration enableAnimationDuration = const Duration(milliseconds: 75);
-  static const Duration positionAnimationDuration = const Duration(milliseconds: 75);
-
-  // Animation controller that is run when interactions occur (taps, drags,
-  // etc.).
-  AnimationController reactionController;
+  static const Duration valueIndicatorAnimationDuration = const Duration(milliseconds: 100);
+
+  // Animation controller that is run when the overlay (a.k.a radial reaction)
+  // is shown in response to user interaction.
+  AnimationController overlayController;
+  // Animation controller that is run when the value indicator is being shown
+  // or hidden.
+  AnimationController valueIndicatorController;
   // Animation controller that is run when enabling/disabling the slider.
   AnimationController enableController;
   // Animation controller that is run when transitioning between one value
   // and the next on a discrete slider.
   AnimationController positionController;
+  Timer interactionTimer;
 
   @override
   void initState() {
     super.initState();
-    reactionController = new AnimationController(
+    overlayController = new AnimationController(
       duration: kRadialReactionDuration,
       vsync: this,
     );
+    valueIndicatorController = new AnimationController(
+      duration: valueIndicatorAnimationDuration,
+      vsync: this,
+    );
     enableController = new AnimationController(
       duration: enableAnimationDuration,
       vsync: this,
     );
     positionController = new AnimationController(
-      duration: positionAnimationDuration,
+      duration: Duration.zero,
       vsync: this,
     );
+    // Create timer in a cancelled state, so that we don't have to
+    // check for null below.
+    interactionTimer = new Timer(Duration.zero, () {});
+    interactionTimer.cancel();
```

Comment:
checking for null is a very minor burden, it's probably better than creating a throwaway timer

Reply:
I meant the coding burden too. It's generally a single question mark whenever you currently do an "isActive" test.